### PR TITLE
fix(docker): reduce config complexity

### LINKED ISSUE #24: Hardcoded MPS port "3000" in docker-compose file
Hardcoded MPS port in docker-compose file and also in MPS Dockerfile.

### PR DESCRIPTION
**Pull request by rsdmike**
closes #24

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-RPS_DB_NAME=postgresdb
+RPS_DB_HOST=db
+RPS_DB_NAME=rpsdb
 RPS_DB_PORT=5432
 RPS_DB_USER=postgresadmin
 RPS_DB_PASSWORD=admin123
 RPS_WEBSOCKETTLS=true
-RPS_WEB_PORT=8081
-RPS_WEBSOCKETPORT=8080
+RPSWEBPORT=8081
```
Comment (brianosburn-intel) [inline .env.template:14]: should these have underlines for separators like the rest?

File: docker-compose.yml [hunk 2]
```diff
@@ -29,40 +29,20 @@ services:
...
     ports: 
-      - "${RPS_WEBSOCKETPORT}:${RPS_WEBSOCKETPORT}"
-      - "${RPS_WEB_PORT}:${RPS_WEB_PORT}"
+      - "${RPSWEBSOCKETPORT}:8080"
+      - "${RPSWEBPORT}:8081"
     depends_on: 
       - mps
+    env_file:
+      - .env
     environment:
```
Comment (brianosburn-intel) [inline docker-compose.yml:38]: where are the environmental variables being imported with this change?

File: .env.template (hunk 1 above)
Comment (rsdmike) [reply .env.template:14]: nope, thats was causing the issue thats linked. Purposefully NOT changing the port inside the container -- thats a no-no.

File: docker-compose.yml [hunk 3]
```diff
@@ -29,40 +29,20 @@ services:
...
-      RPS_CREDENTIALS_PATH: ${RPS_CREDENTIALS_PATH}
       RPS_USER: ${RPS_MPS_USER}
       RPS_PASSWORD: ${RPS_MPS_PASSWORD}
-      RPS_AMT_USER: ${RPS_AMT_USER}
-      RPS_WEB_TLS_CERT: ${RPS_WEB_TLS_CERT}
-      RPS_WEB_TLS_CERT_KEY: ${RPS_WEB_TLS_CERT_KEY}
-      RPS_ROOT_CA_CERT: ${RPS_ROOT_CA_CERT}
-      RPS_LOG_LEVEL: ${RPS_LOG_LEVEL}
-      RPS_CORS_ORIGIN: http://${MPS_COMMON_NAME}:3001
+      RPS_CORS_ORIGIN: http://192.168.1.34:3001
```
Comment (brianosburn-intel) [inline docker-compose.yml:41]: should this be a variable?

File: docker-compose.yml (hunk 2 above)
Comment (rsdmike) [reply docker-compose.yml:38]: the .env file when you run docker-compose up. You can run `docker-compose config`  to see the configured values

File: docker-compose.yml (hunk 3 above)
Comment (rsdmike) [reply docker-compose.yml:41]: WHOOPS

**Review by brianosburn-intel — APPROVED** [review]

**Pull request merged**